package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "slc run",
     "test": "mocha -R spec test",
-    "pretest": "jshint ."
+    "posttest": "eslint ."
```

Comment:
Actually for all Loopback repos we use `posttest`:
```
Modify package.json and/or Gruntfile to ensure that linter (eslint) is run after the tests pass. This typically means adding a "posttest": "eslint" script. If the original package contains a pretest script to run the linter, then remove the pretest script in favor of posttest.
```
Source: https://github.com/strongloop/loopback.io/blob/gh-pages/pages/en/contrib/eslint-guide.md